README.md

```diff
@@ -13,3 +13,23 @@ The *tutorials* folder contains walkthroughs which demonstrate basic Certifai fu
 ## Certifai Documentation
 
 Refer to the Cortex Certifai documentation: https://cognitivescale.github.io/cortex-certifai/docs/about for detailed information about Cortex Certifai.
+
+## Notebooks Table of Contents
+
+| Notebook Name | Description | Task Type | Model Types | Evaluation Types | Key Properties |
+| --- | --- | --- | --- | --- | --- |
+| [AnalyzeFeatureUsage.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/analyze_feature_usage) - Analyzing Feature Usage | Illustrates the use of Certifai to create and run an explanations scan of a locally defined model from first principles. It then analyzes and displays feature distribution of the counterfactuals.  |  Binary classification | Decision Tree <br /> Logistic Regression | Explanations |  Feature Occurrence <br /> Frequency by Model |
+| [BringInYourOwnModel.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/tutorials/bringing_in_your_own_model/part_one) - Building a scan programmatically with your own model | Part one of a tutorial that takes data scientists through model preparation and scan. This notebook/tutorial is also packaged as part of the toolkit and can be accessed at `examples/notebooks/BringingInYourOwnModel.ipynb`. | Binary Classification | SVM <br /> Logistic Regression | Fairness  <br /> Explainability <br />  Robustness |   |
+| (certifai_multiclass_evaluation)[model_train_part1.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates how to start the multiclass classification task type scan. | Multiclass Classification | Logistic Regression | Performance Metric: Accuracy  <br /> Explainability-Explanations <br /> Fairness (by feature) | One-hot encoding <br /> Target encoded features |  
+| [certifai_multiclass_evaluation_part2.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates the second part of the multiclass classification task type. | Multiclass Classification | Logistic Regression | Performance Metric: Accuracy <br /> Explainability-Explanations <br /> Fairness (by feature) | One-hot encoding <br /> Target encoded features |
+| [CertifaiSageMakerExample.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/sagemaker) - Running Cortex Certifai Scan on Sklearn model built and deployed on AWS Sagemaker using Certifai Model Connectors | Uses ml.m4.xlarge Sagemaker notebook instance to create sklearn models to classify german credit loan risk (predict whether loan will be granted or not) and evaluate fairness. | Binary Classification | SVM <br /> Logistic Regression | Fairness | Sagemaker |
```

Comment:
I kinda think that the text next to the notebook name should be part of the description. Something like: "Illustrates running a fairness scan on Sklearn models built with and deployed on AWS Sagemaker using Certifai Model Connectors. The models classify german credit loan risk (whether a loan will be granted or not) and are evaluated for fairness."  - just my 2 cents

Reply:
@laguirre-cs I wanted to tie the repo names with the titles at the tops of the notebooks because often they are quite different.